l10n_br_nfse/models/document_line.py

```diff
@@ -103,18 +103,18 @@ def prepare_line_servico(self):
         return {
             'valor_servicos': float(self.fiscal_price),
             'valor_deducoes': float(self.fiscal_deductions_value),
-            'valor_pis': float(self.pis_value),
-            'valor_cofins': float(self.cofins_value),
-            'valor_inss': float(self.inss_value),
-            'valor_ir': float(self.irpj_value),
-            'valor_csll': float(self.csll_value),
+            'valor_pis': float(self.pis_wh_value),
+            'valor_cofins': float(self.cofins_wh_value),
+            'valor_inss': float(self.inss_wh_value),
+            'valor_ir': float(self.irpj_wh_value),
+            'valor_csll': float(self.csll_wh_value),
```

Comment:
@marcelsavegnago não seria melhor criar entradas no dict como:

![image](https://user-images.githubusercontent.com/1975978/103088312-354e5080-45c9-11eb-849a-ba4ae7be9ea5.png)

Reply:
No caso seria por exemplo ter o valor_pis e o valor_pis_retido?

Reply:
@mileo olhando o fork da trust e outros leiautes de NFSe me pareceu que sempre que for informado o valor do pis, valor do cofins e afins os mesmos serão tratados como retidos. Ou estou enganado ? 

cc @jpcarassato

Reply:
@mileo eu não entendi muito tua sugestão. Pode me ajudar ?

Reply:
@mileo dá uma olhada.. acho que agora ficou melhor.